scripts/06065-Adebayo.js

```diff
@@ -0,0 +1,7 @@
+let member = "Adebayo Ilerioluwa";
+let ID = "HNG-06065";
+let lang = "JavaScript"
+
+let task = `Hello World this is ${member} with HNGi7 ID ${ID} using ${lang} for stage 2 task.`;
```

Comment:
Be sure to include your email after the word `task` as specified [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
thanks, fix requested changes